README.md

```diff
@@ -267,11 +331,16 @@ Branch names should be in the following format:
 
 **Description:** A short description of the branch. This should be in lowercase and use dashes instead of spaces.
 
-## Commit Messages
+**Example:** `feat/issue-66/remove-jedi`
+
+**NOTE:** The linter for this schema prevents you from committing changes during a rebase.
+In this case, use the `--no-verify` flag to make your commit.
+
+## Commit Message Schema
 
 Commit messages should be in the following format:
 
-`<type>(<scope>): <description>`
+`<type>: <description>`
```

Comment:
Why are we changing the format here?

Reply:
The regex as it was didn't leave the scope as optional, but required it. I reached out about it in the general-backend discord channel. @esparr preferred removing it so the commit message wasn't clogged. What are your thoughts?